### PR TITLE
Update protobuf Makefile and re-make go file

### PR DESCRIPTION
**Pull request by mheiber**
- Update Makefile for protocol buffers so it runs
  - Update Makefile to point to the right test file
  - Add ${GOPATH} to --proto_path arg to protoc
  - Update the path to the import for gogoproto (Google code to gh)
- Re-generate go file for protobufs so it has correct comments.
  - Now it says (correctly) that it is generated from `ipld.proto`
  - Update testfile
  - the last test was failing "decoded bytes != encoded bytes"

### CODE REVIEW

**Comment by jbenet** [comment]
Thanks sorry for the delay, we've been busy with other things. @mildred PTAL?

File: coding/pb/Makefile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 PB = $(wildcard *.proto)
 GO = $(PB:.proto=.pb.go)
 
-all: $(GO) test.pb
+all: $(GO) testfile
 
 %.pb.go: %.proto
-	protoc --gogo_out=. --proto_path=../../../../../../:/usr/local/opt/protobuf/include:. $<
+	protoc --gogo_out=. --proto_path=${GOPATH}/src:. --proto_path=/usr/local/opt/protobuf241/include:. $<
```
Comment (jbenet) [inline coding/pb/Makefile:7]: this LGTM, but can someone else verify?

**Comment by mildred** [comment]
It build for me. LGTM.

See also https://github.com/ipfs/go-ipld/pull/19/commits/3d1a32e0e8d749e91d09d0a5164857be4f191172 that rework the pb package because the Makefile is no longer working. This doesn't prevent this PR to be merged however

File: coding/pb/Makefile (hunk 1 above)
Comment (parkan) [inline coding/pb/Makefile:7]: this looks like protobuf was installed from `homebrew-versions` on the building machine, `/usr/local/opt/protobuf` is definitely the standard path

**Comment by awalterschulze** [comment]
You could possibly also simplify your protos.

These proto files are doing the same thing
https://github.com/ipfs/go-ipfs/blob/2b6da0c024f28abeb16947fb452787196a6b56a2/merkledag/pb/merkledag.proto
https://github.com/ipfs/go-ipld/blob/fd22efe0985ec6ce4b49f6621016176710345d8e/coding/pb/ipld.proto
Both need to import gogoproto.proto and enable all the options.

You could maybe use something like gogoslick_out instead of gogo_out
https://github.com/gogo/protobuf/blob/master/Readme.md#more-speed-and-more-generated-code

**Pull request closed without merging**